core/metric.go

```diff
@@ -64,6 +67,31 @@ func (n Namespace) Strings() []string {
 	return ns
 }
 
+// getSeparator returns an valid separator from nsPriorityList.
+// Otherwise the core separator is returned.
+func (n Namespace) getSeparator() string {
```

Comment:
Just thought someone could want to use that for generating a task manifest dynamically. (not a relevant use case)